src/components/users/UserLoginForm.js

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from 'react'
+import emailValidator from 'email-validator'
+
+import RaisedButton from 'material-ui/RaisedButton'
+
+import TextField from './TextFieldExtended'
+
+class UserLoginForm extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      password: null,
+      email: null,
+
+      validFields: {
+        password: false,
+        email: false
+      }
+    }
+  }
+
+  onFieldValueChange = (fieldName, fieldValue, hasError) => {
+    let newState = Object.assign({}, this.state)
+    newState.validFields[fieldName] = !hasError
+    newState[fieldName] = fieldValue
+    this.setState(newState)
+  }
+
+  handleSubmit = event => {
+    console.log(
+            'not implemented yet i have',
+            this.state,
+            'maybe in future i will figure out what can i do with all those'
+        )
+  }
+
+  render () {
+    return (
+      <div style={main}>
+        <h2>Login</h2>
+        <TextField
+          name='email'
+          hintText='Email'
+          floatingLabelText='Email'
+          validationFn={value => {
```

Comment:
You can shorten this function by using && operator and single line arrow function:

`value => (!emailValidator.validate(value) && "Wrong email")`